wgpu-core/src/device/mod.rs

```diff
@@ -948,6 +948,15 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             None => (),
         };
 
+        let texture_features = conv::texture_features(desc.format);
+        if texture_features != wgt::Features::empty() && !device.features.contains(texture_features)
```

Comment:
nit: is the first check necessary?

Reply:
I'm really not sure, I added it out of an abundance of causion as I know bitflags does weird things with empty flags. If you think it's not I can remove it, I just wasn't sure at the time.